x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleType.java

```diff
@@ -58,6 +59,9 @@
 
     static final Set<String> HOT_ACTIONS_THAT_REQUIRE_ROLLOVER = Sets.newHashSet(ReadOnlyAction.NAME, ShrinkAction.NAME,
         ForceMergeAction.NAME);
```

Comment:
We need to add `SearchableSnapshotAction.NAME` to this list, because we should always ensure that a `rollover` action is used in the `hot` phase if searchable snapshots are specified